datashader/reductions.py

```diff
@@ -1793,32 +1818,34 @@ def _build_combine(self, dshape, antialias, cuda, partitioned):
         invalid = isminus1 if self.selector.uses_row_index(cuda, partitioned) else isnull
 
         @ngjit
-        def combine_cpu_2d(aggs, selector_aggs):
-            ny, nx = aggs[0].shape
+        def combine_cpu(aggs, selector_aggs):
```

Comment:
There is a lot of similar but not quite identical code here that I am planning to refactor in a separate PR.